libs/email/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Effect } from 'effect'
```

Comment:
Why is there still Effect? Please remove it completely, including from `package.json`.